util/httputil/context.go

```diff
@@ -33,20 +33,24 @@ func ContextWithPath(ctx context.Context, path string) context.Context {
 
 // ContextFromRequest returns a new context from a requests with identifiers of
 // the request to be used later when logging the query.
-func ContextFromRequest(ctx context.Context, r *http.Request) (context.Context, error) {
-	ip, _, err := net.SplitHostPort(r.RemoteAddr)
-	if err != nil {
-		return ctx, err
+func ContextFromRequest(ctx context.Context, r *http.Request) context.Context {
+	m := map[string]string{
```

Comment:
why `m`? `reqCtxVal` maybe to be more explicit?

Reply:
Given how small the function is, `m` looked enough to me, but I can change it to `reqCtxVal` as well.